region/maridia/outer.json

```diff
@@ -836,6 +840,11 @@
                     "canRiskPermanentLossOfAccess",
                     "canSuitlessMaridia",
                     "canTrickyJump"
+                  ],
+                  "note": [
+                    "Jump as the first action when breaking the tube to gravity jump to the top of the room.",
+                    "Try to end G-Mode before the Power Bomb finished detonating.",
+                    "This can only be attempted once."
```

Comment:
```suggestion
                    "Jump as the first action after breaking the tube to gravity jump to the top of the room.",
                    "The tube will break when exiting G-Mode or when the Power Bomb finishes detonating, whichever is later.",
                    "Open the door and go through it during the ascent. This can only be attempted once."
```